packages/react/src/utilities/decorators/withViewport.tsx

```diff
@@ -44,6 +44,11 @@ export interface IWithViewportProps {
    * direct DOM dependencies.
    */
   skipViewportMeasures?: boolean;
+  /**
```

Comment:
```suggestion
  /**
   * Whether or not `withViewport` will delay before first measuring the viewport size. 
   * Setting this will delay measurement by the same amount as the debounce for resizing the window.
   * This is useful for giving the child of the viewport time to render before measuring.
   * @default false
   */
  delayFirstMeasure?: boolean;
```